editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -138,21 +138,27 @@ export function SectionsOverviewList({
       {Object.entries(sectionNodes).map(([sectionId, sectionNode]) => (
         <React.Fragment key={sectionId}>
           <dt>
-            {showChange &&
-            sectionStatuses[sectionId] === SectionStatus.Completed ? (
-              <Link
-                onClick={() => changeFirstAnswerInSection(sectionId)}
-                component="button"
-                sx={{ fontFamily: "inherit", fontSize: "inherit" }}
-              >
-                {sectionNode.data.title}
-                <span style={visuallyHidden}>
-                  {`Change ${sectionNode.data.title}`}
-                </span>
-              </Link>
-            ) : (
-              sectionNode.data.title
-            )}
+            <Typography variant="body1" component="div">
+              {showChange &&
+              sectionStatuses[sectionId] === SectionStatus.Completed ? (
+                <Link
+                  onClick={() => changeFirstAnswerInSection(sectionId)}
+                  component="button"
+                  sx={{
+                    fontFamily: "inherit",
+                    fontSize: "inherit",
+                    textAlign: "left",
+                  }}
+                >
+                  {sectionNode.data.title}
```

Comment:
```suggestion
            <Typography variant="body2">
                  {sectionNode.data.title}
            </Typography>
```

This approach (`Link` wrapping a `Typography`) is more consistent with the code base and allows you to drop the `Link`'s `sx` prop entirely (the awkward workaround to inherit font details).

Appreciate this isn't your change here but seems worth tidying up 👍